website/docs/error-reference.md

```diff
@@ -936,6 +934,19 @@ srb tc --packager-layers util,lib,app
 If the flag is passed with no argument, then the default valid layers are
 `library` and `application`.
 
+## 3726
+
+> This error is specific to Stripe's custom `--stripe-packages` mode. If you are
+> at Stripe, please see [go/modularity](http://go/modularity) and
+> [go/layers](http://go/layers) for more.
+
+If a package is at `strict_dependencies` level `layered` or stricter, there are
+2 restrictions on what packages it may import:
+
+- all packages it imports must also be at `strict_dependencies` level `layered`
+  (or stricter)
+- all packages must be in the same or lower layer
```

Comment:
```suggestion
- all packages it imports must be in the same or lower `layer`
```

Reply:
I might further clarify this with an example, like "For example, given `--packager-layers util,lib,app`, all imports for a package with `layer 'lib'` must either also be `layer 'lib'` or be `layer 'util'`."